actors/builtin/miner/miner_test.go

```diff
@@ -1384,15 +1513,34 @@ func (h *actorHarness) submitWindowPost(rt *mock.Runtime, deadline *miner.Deadli
 		rt.ExpectGetRandomness(crypto.DomainSeparationTag_WindowedPoStChallengeSeed, deadline.Challenge, buf.Bytes(), abi.Randomness(challengeRand))
 	}
 	{
+		var goodInfo *miner.SectorOnChainInfo
```

Comment:
A brief explanation of intent, please. "Find the first non-faulty sector in the post, which is to be substituted for all the skipped fault sector infos".